### PR TITLE
Update change log for v3 + rc.1 release

### PR DESCRIPTION
**Pull request by axunonb**
@304NotModified @NKnusperer 
Anything relevant missing, which is already in the master branch?

### CODE REVIEW

**Comment by 304NotModified** [comment]
Will check this week :)

**Review by 304NotModified — DISMISSED** [review]
Looks great! Made some small changes. See commits

**Comment by NKnusperer** [comment]
The `DemoApplication` is now a `net5` app, not sure if this is worth mentioning because it does not effect the library.

**Comment by axunonb** [comment]
Anything more to happen before publishing the v3 release?

**Comment by 304NotModified** [comment]
nope! Good to go! Maybe a RC first? To be sure?

File: appveyor.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ for:
     build_script:
       - ps: dotnet restore --verbosity quiet
       - ps: |
-          $version = "3.0.0-alpha.2"
+          $version = "3.0.0"
```
Comment (304NotModified) [inline appveyor.yml:24]:
```suggestion
          $version = "3.0.0-rc1"
```

File: appveyor.yml [hunk 2]
```diff
@@ -21,7 +21,7 @@ for:
     build_script:
       - ps: dotnet restore --verbosity quiet
       - ps: |
-          $version = "3.0.0-alpha.2"
+          $version = "3.0.0-rc1"
```
Comment (304NotModified) [inline appveyor.yml:24]:
```suggestion
          $version = "3.0.0-rc.1"
```

**Comment by 304NotModified** [comment]
Ow I think rc.1 is better, according to semver.org and also more consistent with alpha.1, alpha.2  etc

**Review by 304NotModified — APPROVED** [review]

**Pull request merged**